README.md

```diff
@@ -1,22 +1,41 @@
-# Welcome to Pocket Portal!
-
-### Run with Remix
+<div align="center">
+<h1>Grove Portal UI</h1>
+<img src="https://storage.googleapis.com/grove-brand-assets/Presskit/Logo%20Joined-2.png" alt="Grove logo" width="500"/>
+</div>
+<br/>
+
+# Table of Contents <!-- omit in toc -->
+
+- [Run with **Remix**](#run-with-remix)
+- [Deployment](#deployment)
+- [Development](#development)
+  - [Env](#env)
+  - [Node Version](#node-version)
+  - [Frontend](#frontend)
+  - [Stripe Webhook Forwarding](#stripe-webhook-forwarding)
+  - [Environment Variables](#environment-variables)
+  - [Backend](#backend)
+
+## Run with **Remix**
 
 - [Remix Docs](https://remix.run/docs)
 
 ## Deployment
 
-### PR / DEV
+1. **Test Locally**
+
+   - Test your changes locally before creating a PR.
 
-Create PR and vercel will create a preview build based on the PR
+2. **PR / STAGE**
 
-### STAGE
+   - Create a PR into the "staging" branch. Continuous Deployment (CD) will automatically deploy to [https://staging.portal.grove.city](https://staging.portal.grove.city).
```

Comment:
```suggestion
   - Create a PR into the `stage` branch.
   - Continuous Deployment (CD) will automatically deploy it to [staging.portal.grove.city](https://staging.portal.grove.city) after it is merged.
   - E.g. <INSERT_LINK>
```

Reply:
What should I insert in the `e.g.` line?

Reply:
Done.